plugin.yaml

```diff
@@ -1,5 +1,5 @@
 name: "s3"
-version: "0.15.1"
```

Comment:
Please don't update the version here. It should be done separately when the release is prepared.

Reply:
Ok